From 3d0e9337a1c75712fed021207118dfb568fd30a6 Mon Sep 17 00:00:00 2001
From: Shivani Raval <sraval@habana.ai>
Date: Tue, 10 Sep 2024 13:45:40 +0300
Subject: [PATCH] [FSW-6362] config: renamed mgmt to mgmt_io

Renamed the config files parameter from mgmt to
mgmt_io. so now we have configs like mgmt_io and
mgmt_compute.

Change-Id: I3beceb28295bef1790a1536442d32974b2851621
Signed-off-by: Shivani Raval <sraval@habana.ai>
---

diff --git a/configs/fs1_arc_asic_defconfig b/configs/fs1_arc_asic_defconfig
index e0a1e83..dc61b10 100644
--- a/configs/fs1_arc_asic_defconfig
+++ b/configs/fs1_arc_asic_defconfig
@@ -16,7 +16,7 @@
 BR2_PACKAGE_BAREMETAL=y
 BR2_PACKAGE_DSM=y
 BR2_PACKAGE_PREBOOT=y
-BR2_PACKAGE_MGMT=y
+BR2_PACKAGE_MGMT_IO=y
 BR2_PACKAGE_MGMT_COMPUTE=y
 BR2_PACKAGE_RESET=y
 BR2_POST_IMAGE_FW_LOADER_ITS_TEMPLATE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/fs1/agent-fw_loader.its"
diff --git a/configs/fs1_arc_defconfig b/configs/fs1_arc_defconfig
index e0a1e83..dc61b10 100644
--- a/configs/fs1_arc_defconfig
+++ b/configs/fs1_arc_defconfig
@@ -16,7 +16,7 @@
 BR2_PACKAGE_BAREMETAL=y
 BR2_PACKAGE_DSM=y
 BR2_PACKAGE_PREBOOT=y
-BR2_PACKAGE_MGMT=y
+BR2_PACKAGE_MGMT_IO=y
 BR2_PACKAGE_MGMT_COMPUTE=y
 BR2_PACKAGE_RESET=y
 BR2_POST_IMAGE_FW_LOADER_ITS_TEMPLATE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/fs1/agent-fw_loader.its"
diff --git a/configs/fs1_arc_onix_defconfig b/configs/fs1_arc_onix_defconfig
index e0a1e83..dc61b10 100644
--- a/configs/fs1_arc_onix_defconfig
+++ b/configs/fs1_arc_onix_defconfig
@@ -16,7 +16,7 @@
 BR2_PACKAGE_BAREMETAL=y
 BR2_PACKAGE_DSM=y
 BR2_PACKAGE_PREBOOT=y
-BR2_PACKAGE_MGMT=y
+BR2_PACKAGE_MGMT_IO=y
 BR2_PACKAGE_MGMT_COMPUTE=y
 BR2_PACKAGE_RESET=y
 BR2_POST_IMAGE_FW_LOADER_ITS_TEMPLATE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/fs1/agent-fw_loader.its"
diff --git a/configs/fs1_arc_pldm_defconfig b/configs/fs1_arc_pldm_defconfig
index e0a1e83..dc61b10 100644
--- a/configs/fs1_arc_pldm_defconfig
+++ b/configs/fs1_arc_pldm_defconfig
@@ -16,7 +16,7 @@
 BR2_PACKAGE_BAREMETAL=y
 BR2_PACKAGE_DSM=y
 BR2_PACKAGE_PREBOOT=y
-BR2_PACKAGE_MGMT=y
+BR2_PACKAGE_MGMT_IO=y
 BR2_PACKAGE_MGMT_COMPUTE=y
 BR2_PACKAGE_RESET=y
 BR2_POST_IMAGE_FW_LOADER_ITS_TEMPLATE="$(BR2_EXTERNAL_HABANALABS_PATH)/board/habanalabs/fs1/agent-fw_loader.its"
